pep-0518.txt

```diff
@@ -375,6 +375,17 @@ An example Python literal file for the proposed data would be::
      }
 
 
+Sticking with ``setup.cfg``
+---------------------------
+
+There are two issues with ``setup.cfg`` used by setuptools as a general
+format. One is that they are ``.ini`` files which have issues as mentioned
+in the configparser_ discussion above. The other is that the schema for
+that file has never been rigourlessly defined and thus it's unknown which
```

Comment:
Do you mean "rigorously" (not sure of the non-U.S. spelling) instead of "rigourlessly"?

Reply:
Fixed

Reply:
> ``.ini`` files which have issues as mentioned
+in the configparser_ discussion above

I LOVE toml. But if we go the parser route, the current toml parser in Python is atrocious. It's brittle and makes syntax error very hard to debug.

> The other is that the schema for
+that file has never been rigourlessly defined

It's an exaggeration. It's not perfect, but it's quite complete and still way easier than to start with setup.py:
 
http://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files

It also had fantastic goodies like including the version of the package in one line.